schemas/process.yml

```diff
@@ -32,6 +32,44 @@
       - at: process
```

Comment:
As part of changes to process.yml, we were planning on dropping the pgid property, but since my process.yml is just deltas i snuck that detail in via a comment.

Reply:
process.group_leader.pid   === process.pgid

process.parent.group_leader.pid === process.parent.pgid

Reply:
I've added a deprecation notice, with guidance on the updated location